packages/react/docs/generated/generated_static_pages.json

```diff
@@ -8,49 +8,13 @@
         "type": "Generic",
         "anchorLink": "setup",
         "title": "Setup",
-        "sectionContent": "\n1. Run one of the example commands to install the package\n1. Import the component, hook, or utility that you want to use in your app. Refer to the list of [what's available](components) in Hydrogen React.\n      ",
+        "sectionContent": "\n1. Run one of the example commands to install the package\n1. Import the component, hook, or utility that you want to use in your app. For more detailed instructions, see the Getting Started Guide.\n      ",
```

Comment:
Can we link to the guide?

Reply:
Already is good sir :)

![image](https://user-images.githubusercontent.com/25570543/216723260-cbe26a51-8dd5-444f-8ead-62e8b9f3b900.png)